src/server/worker/driver/driver.go

```diff
@@ -17,6 +16,8 @@ import (
 	"syscall"
 	"time"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I'm not sure your IDE should have moved `metav1` up into its own section.  This is a weird goland-ism; goimports and gopls won't touch it; it assumes it's where you want it, but goland always does weird things like this.  Not a big deal and you don't have to change it, but if it's some setting that can be configured, it would be good to configure it.

I think we *aim* for sectioning imports by stdlib, pachyderm, and third-pary, but most reliably mix everything except stdlib into one paragraph.